### PR TITLE
Always have inviters first in followers list

### PR DESCRIPTION
**Pull request by stkem**
Order is arbitrary if there are multiple inviters
@andrewconner @ahsu1230

### CODE REVIEW

**Comment by andrewconner** [comment]
Looks good, thanks!

**Comment by ahsu1230** [comment]
awesome! tyty

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryCommander.scala [hunk 1]
```diff
@@ -193,7 +193,16 @@ class LibraryCommander @Inject() (
 
     val followerInfosByLibraryId = libraries.map { library =>
       val (collaborators, followers, _, counts) = getLibraryMembers(library.id.get, 0, maxMembersShown, fillInWithInvites = false)
-      library.id.get -> ((collaborators ++ followers).map(_.userId), counts)
+      val inviters: Seq[LibraryMembership] = viewerUserIdOpt.map { userId =>
+        db.readOnlyReplica { implicit session =>
+          libraryInviteRepo.getWithLibraryIdAndUserId(library.id.get, userId).filter { invite =>
```
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryCommander.scala:198]: if you prefer, you could use `filterNot` in this case

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryCommander.scala [hunk 2]
```diff
@@ -193,7 +193,16 @@ class LibraryCommander @Inject() (
...
+            invite.inviterId != library.ownerId
+          }.map { invite =>
+            libraryMembershipRepo.getWithLibraryIdAndUserId(library.id.get, invite.inviterId)
+          }
+        }.flatten
+      }.getOrElse(Seq.empty)
+      library.id.get -> ((inviters ++ collaborators.filter(!inviters.contains(_)) ++ followers.filter(!inviters.contains(_))).map(_.userId), counts)
```
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryCommander.scala:205]: same here if you think it's more readable =)

**Pull request merged**